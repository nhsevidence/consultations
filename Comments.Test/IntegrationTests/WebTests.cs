using Comments.Test.Infrastructure;
using System.Threading.Tasks;
using Xunit;

namespace Comments.Test.IntegrationTests
{
    public class WebTests : TestBase
    {
        [Fact]
        public async Task Get_Consultations_Homepage()
        {
            // Act
            var response = await _client.GetAsync("/consultations"); //this route is temporary.
            response.EnsureSuccessStatusCode();

            var responseString = await response.Content.ReadAsStringAsync();

            // Assert
            responseString.ShouldMatchApproved();
        }

        [Fact]
        public async Task Get_React_Document_Page()
        {
            // Act
            var response = await _client.GetAsync("/consultations/1/1/introduction");
            response.EnsureSuccessStatusCode();

            var responseString = await response.Content.ReadAsStringAsync();

            // Assert
            responseString.ShouldMatchApproved();
        }
    }
}
