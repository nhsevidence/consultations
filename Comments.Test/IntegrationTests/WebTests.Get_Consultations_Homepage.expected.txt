
<!DOCTYPE html>
  <html>
    <head>
      <meta name="viewport" content="width=device-width" />
      <title>
      </title>
    </head>
    <body>
      <div>
        todo: tophat
      </div>
      <div>
        <h1>
          consultations hub page goes here maybe.
        </h1>
      </div>
      <div>
        todo: footer
      </div>
    </body>
  </html>